apps/home/templates/apps_home/overview_page.html

```diff
@@ -6,7 +6,8 @@
 <div class="fullwidth">
     <div id="layout-grid">
     <section id="layout-grid__area--herounit">
-        {% if page.intro_image %}
+        {{ request.LANGUAGE_CODE }}
+        {% if page.intro_image and request.LANGUAGE_CODE != 'de-ls' %}
```

Comment:
why use a variable and template tag here and just a template tag above, do I miss something?
also the language switch doesn't seem to work on overview anymore, does on gruenbuch though, maybe doing it other was is better?

Reply:
@khamui What is the status here? Did you resolve it, can I have another look? And do you also want to add it to the microsites or did you already?

Reply:
yes @fuzzylogic2000 it is resolved and works as expected.